host-interaction/file-system/read/read-initialization-file.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: read initialization file
+    namespace: host-interaction/file-system/read
+    author: "@_re_fox"
+    scope: function
+  examples:
```

Comment:
examples needs to be under the `rule.meta` block

if you have the source of capa checked out, then you can run `python scripts/linter.py rules` to find quick fixes like this

Reply:
Ah, silly mistake.  Fixed.